MODULE.bazel

```
@@ -0,0 +1,21 @@
+module(
+    name = "xacro",
+    version = "2.0.11",
+)
+
+bazel_dep(name = "rules_license", version = "1.0.0")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "rules_python", version = "0.40.0")
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+python.toolchain(
+    python_version = "3.11",
+)
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+pip.parse(
+    hub_name = "pypi",
+    python_version = "3.11",
+    requirements_lock = "//bazel:requirements_lock.txt",
+)
+use_repo(pip, "pypi")
```

The repo name needs to be unique among all modules. So using `pypi` is not a good choice. Prefer something like `xacro_python_dependencies`.
```suggestion
    hub_name = "xacro_python_dependencies",
    python_version = "3.11",
    requirements_lock = "//bazel:requirements_lock.txt",
)
use_repo(pip, "xacro_python_dependencies")
```

---

Good catch.